### PR TITLE
Integrate pomodoro and video

### PR DESCRIPTION
**Pull request by chaule-mck**

### CODE REVIEW

File: pho/src/pages/space.tsx [hunk 1]
```diff
@@ -36,13 +41,38 @@ const SpacePage: React.FC<{}> = () => {
...
+        style={isVideoVisible ? {} : { display: 'none' }}
+      >
+        <div className="location-text">{name}</div>
+      </div>
+
+      <div
+        style={isVideoVisible ? { position: 'absolute' } : { display: 'none' }}
+      >
+        <Pomodoro
+          timestamp={new Date(startTime ?? '').getTime()}
```
Comment (chaule-mck) [inline pho/src/pages/space.tsx]: @richardtlp please help me take a look at this part. thankiu

File: pho/src/pages/Space.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.location-name-box {
+  box-sizing: border-box;
+
```
Comment (richardtlp) [inline pho/src/pages/Space.css]: I think we should rename this file to `space.css`

File: pho/src/pages/space.tsx (hunk 1 above)
Comment (richardtlp) [reply pho/src/pages/space.tsx]: It's OK for now. Let me check the Pomodoro component implementation again with @thienanhvu.

**Review by richardtlp — APPROVED** [review]

File: pho/src/pages/space.tsx [hunk 3]
```diff
@@ -36,13 +41,38 @@ const SpacePage: React.FC<{}> = () => {
...
+  const [isVideoVisible, setIsVideoVisible] = useState(false);
+
   return (
     <div>
-      <Heading>
-        Space #{id} - {name}
-      </Heading>
-      <p>#Bg-Video-Component</p>
-      <p>Socket ID: {socket.id}</p>
+      <div
```
Comment (minhnguyenphamquoc) [inline pho/src/pages/space.tsx]: Should have changed to:
```jsx
     {isVideoVisible && <div>...<div>}
```
@chaule-hub

**Review by richardtlp — APPROVED** [review]

**Pull request merged**